google/cloud/firestore_v1/async_aggregation.py

```diff
@@ -25,14 +25,14 @@
 
 from typing import AsyncGenerator, List, Optional, Union
 
-
+from google.cloud.firestore_v1.async_stream_generator import AsyncStreamGenerator
 from google.cloud.firestore_v1.base_aggregation import (
     AggregationResult,
     _query_response_to_result,
     BaseAggregationQuery,
 )
-
-from google.cloud.firestore_v1 import async_stream_iterator, transaction
+from google.cloud.firestore_v1.base_document import DocumentSnapshot
```

Comment:
Is this just used for type annotations? If so, you can put it in an [`if TYPE_CHECKING` block](https://github.com/googleapis/python-bigtable/blob/c573e9b695658f3e0c78eb3443d0cf26fea57da9/google/cloud/bigtable/data/_async/_read_rows.py#L42) to avoid importing it in production